Project.toml

```
@@ -40,6 +46,7 @@ DataStructures = "0.18"
 Dictionaries = "0.3.15"
 Distributions = "0.25.86"
 DocStringExtensions = "0.8, 0.9"
+EinExprs = "0.6"
```

```suggestion
EinExprs = "0.6.4"
```
when the `KaHyPar` extension was added.